Cargo.lock

```diff
@@ -1739,9 +1739,18 @@ dependencies = [
 
 [[package]]
 name = "tinyvec"
-version = "0.3.4"
+version = "1.0.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "b78a366903f506d2ad52ca8dc552102ffdd3e937ba8a227f024dc1d1eae28575"
+dependencies = [
+ "tinyvec_macros",
```

Comment:
urgh, I wish url didn't swap tinyvec for smallvec :(

I also wish Microsoft refactored LSP typescript so that we can generate lsp support without depending on lsp-types and Url...

Reply:
Meeee too.